mozci/task.py

```diff
@@ -362,6 +362,12 @@ def configuration(self):
 # Don't perform type checking because of https://github.com/python/mypy/issues/5374.
 @dataclass  # type: ignore
 class RunnableSummary(ABC):
+    @property
+    def is_intermittent(self):
+        return self.status == Status.INTERMITTENT or any(
+            c == "intermittent" for c, n in self.classifications
```

Comment:
There's also an `autoclassified intermittent` classification.. not sure how much it actually gets used (if ever), but might as well support it I think.

Reply:
Is it set automatically or is it still chosen by sheriffs?

Reply:
(I'm wondering if we can trust it as we can trust `intermittent`)

Reply:
I've been told sheriffs don't use it anyway, so I'll not add it. Perhaps we can remove it from Treeherder options? CC @camd